web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss

```diff
@@ -1126,17 +1127,17 @@ button#next.usa-button,
   text-align: center;
   float: none;
   height: 38px;
-  margin: 3px 0 0;
+  margin: 3px 10px 0;
```

Comment:
Please consider deleting this instead of commenting it out. We can always go back in the git history if we want to retrieve something. (I realize it might help to have it there for reference for awhile.)